### DISCUSSION TITLE
v2.8.0

### DISCUSSION THREAD

**Discussion by timmo001** [timmo001/system-bridge#1136] [General]
## ✨ New features

- Replace Tray with Python Qt GUI @timmo001 (#1061)
- Send to another bridge (GUI) @timmo001 (#1098)
- Send to Bridge Setup (GUI) - Manual @timmo001 (#1120)
- Observer - Only observe requested and specific data from websocket request @timmo001 (#1119)

## 🚀 Enhancements

- Split processes into multiple endpoints @timmo001 (#1118)
- App Improvements @timmo001 (#1124)
- Split Endpoints @timmo001 (#1125)

## 🧰 Maintenance

- Use shared workflows @timmo001 (#1054)
- Add Devcontainer @timmo001 (#1121)

## ⬆️ Dependency updates

- Bump typeorm from 0.2.38 to 0.2.39 @dependabot (#1055)
- Bump systeminformation from 5.9.9 to 5.9.10 @dependabot (#1059)
- Bump typeorm from 0.2.39 to 0.2.40 @dependabot (#1058)
- Bump eslint-plugin-import from 2.25.2 to 2.25.3 in /frontend @dependabot (#1057)
- Bump eslint-plugin-import from 2.25.2 to 2.25.3 @dependabot (#1056)
- Bump systeminformation from 5.9.10 to 5.9.11 @dependabot (#1060)
- Bump @mdi/js from 6.4.95 to 6.5.95 in /frontend @dependabot (#1067)
- Bump concurrently from 6.3.0 to 6.4.0 @dependabot (#1068)
- Bump systeminformation from 5.9.11 to 5.9.13 @dependabot (#1069)
- Bump @types/react from 17.0.34 to 17.0.35 @dependabot (#1074)
- Bump @nestjs/websockets from 8.2.0 to 8.2.1 @dependabot (#1072)
- Bump @nestjs/platform-express from 8.2.0 to 8.2.1 @dependabot (#1073)
- Bump @nestjs/common from 8.2.0 to 8.2.1 @dependabot (#1071)
- Bump @nestjs/testing from 8.2.0 to 8.2.1 @dependabot (#1076)
- Bump @types/react from 17.0.34 to 17.0.35 in /frontend @dependabot (#1078)
- Bump @nestjs/platform-ws from 8.2.0 to 8.2.1 @dependabot (#1077)
- Bump @nestjs/schematics from 8.0.4 to 8.0.5 @dependabot (#1075)
- Bump @nestjs/core from 8.2.0 to 8.2.1 @dependabot (#1070)
- Bump typescript from 4.4.4 to 4.5.2 in /frontend @dependabot (#1091)
- Bump @nestjs/websockets from 8.2.1 to 8.2.2 @dependabot (#1090)
- Bump @nestjs/swagger from 5.1.4 to 5.1.5 @dependabot (#1089)
- Bump @nestjs/cli from 8.1.4 to 8.1.5 @dependabot (#1088)
- Bump @nestjs/common from 8.2.1 to 8.2.2 @dependabot (#1082)
- Bump @nestjs/testing from 8.2.1 to 8.2.2 @dependabot (#1084)
- Bump @types/uuid from 8.3.1 to 8.3.3 @dependabot (#1086)
- Bump @types/node from 16.11.7 to 16.11.9 @dependabot (#1095)
- Bump @types/jest from 27.0.2 to 27.0.3 @dependabot (#1094)
- Bump typeorm from 0.2.40 to 0.2.41 @dependabot (#1087)
- Bump @nestjs/platform-express from 8.2.1 to 8.2.2 @dependabot (#1085)
- Bump typescript from 4.4.4 to 4.5.2 @dependabot (#1083)
- Bump @nestjs/core from 8.2.1 to 8.2.2 @dependabot (#1081)
- Bump @nestjs/platform-ws from 8.2.1 to 8.2.2 @dependabot (#1080)
- Bump systeminformation from 5.9.13 to 5.9.15 @dependabot (#1096)
- Bump better-sqlite3 from 7.4.4 to 7.4.5 @dependabot (#1097)
- Bump tsconfig-paths from 3.11.0 to 3.12.0 @dependabot (#1099)
- Bump @types/react from 17.0.35 to 17.0.36 @dependabot (#1105)
- Bump @types/react from 17.0.35 to 17.0.36 in /frontend @dependabot (#1113)
- Bump @nestjs/core from 8.2.2 to 8.2.3 @dependabot (#1112)
- Bump @nestjs/testing from 8.2.2 to 8.2.3 @dependabot (#1110)
- Bump @nestjs/platform-ws from 8.2.2 to 8.2.3 @dependabot (#1107)
- Bump @nestjs/common from 8.2.2 to 8.2.3 @dependabot (#1111)
- Bump @nestjs/platform-express from 8.2.2 to 8.2.3 @dependabot (#1108)
- Bump @nestjs/websockets from 8.2.2 to 8.2.3 @dependabot (#1109)
- Bump ws from 7.5.5 to 7.5.6 @dependabot (#1116)
- Bump @types/node from 16.11.9 to 16.11.10 @dependabot (#1115)
- Bump @types/react from 17.0.36 to 17.0.37 @dependabot (#1117)
- Bump @types/react from 17.0.36 to 17.0.37 in /frontend @dependabot (#1123)
- Bump prettier from 2.4.1 to 2.5.0 @dependabot (#1122)
- Bump pkg from 5.4.1 to 5.5.1 @dependabot (#1126)
- Bump jest from 27.3.1 to 27.4.0 @dependabot (#1127)
- Bump @types/node from 16.11.10 to 16.11.11 @dependabot (#1129)
- Bump jest from 27.4.0 to 27.4.1 @dependabot (#1128)
- Bump jest from 27.4.1 to 27.4.2 @dependabot (#1130)
- Bump jest from 27.4.2 to 27.4.3 @dependabot (#1135)

<hr /><em>This discussion was created from the release <a href='https://github.com/timmo001/system-bridge/releases/tag/v2.8.0'>v2.8.0</a>.</em>

**Comment by julianrinaldi** [comment]
How is the GUI accessed now that there is no longer a system tray icon on Linux?
